x/evm/keeper/keeper.go

```diff
@@ -301,3 +302,36 @@ func (k Keeper) BaseFee(ctx sdk.Context, ethCfg *params.ChainConfig) *big.Int {
 	}
 	return baseFee
 }
+
+// ResetTransientGasUsed reset gas used to prepare for execution of current cosmos tx, called in ante handler.
+func (k Keeper) ResetTransientGasUsed(ctx sdk.Context) {
+	store := ctx.TransientStore(k.transientKey)
+	store.Delete(types.KeyPrefixTransientGasUsed)
+}
+
+// GetTransientGasUsed returns the gas used by current cosmos tx.
+func (k Keeper) GetTransientGasUsed(ctx sdk.Context) uint64 {
```

Comment:
can you update the spec with the transient gas used key 🙏

Reply:
done